doc/navigating.rst

```diff
@@ -38,116 +39,59 @@ so you should click it to see what it has to tell you.
-    defined multiple synchronization targets, the window displays each local 
-    folder.
-* **Disk space indicator**: Indicates the amount of space currently used on the 
-  server.
-* **Operation indicator**: Displays the status of the current synchronization 
-  process or indicates ``Up to date`` if the server and client are in sync.
-* **Recent Changes**: Displays the last six files modified, and shows the 
-  current synchronization status listing all changes since the last restart of the 
-  ownCloud client.
-* ``Settings...``: The settings menu.
-* ``Help``: Opens a browser to display ownCloud Desktop Client Guide.
-* ``Sign out``: Stops synchronization.
-* ``Quit ownCloud``: Quits the ownCloud Client, ending any currently running
-  synchronizations.
+* Open ownCloud in browser
+* Managed folder
+* Open folder [your local sync folder]
+* Up to date
+* Recent changes
+* Settings
+* Help
+* Sign out
+* Quit ownCloud
 
 Using the Account Settings Window
 ---------------------------------
 
 .. index:: account settings, user, password, Server URL
 
-The ``Account`` window provides a summary for your ownCloud account settings.  
-You can manage which folders and files you want to synchronize, change your 
-account settings, and pause and resume synchronization.
-
-To access and modify the account settings:
-
-.. image:: images/settings_account.png
-   :scale: 50 %
-
-The fields and options in this window include:
-
-* ``Connected to <ownCloud instance> as <user>`` field:  Your user account on an ownCloud 
-  server.
-
-* ``Add Folder...`` button: Add another folder to the 
-  synchronization process (see ``Adding a Folder``).
-
-* ``Pause/Resume`` button: Pauses the current sync (or prevents the client from 
-  starting a new sync), or resumes the sync process.
-
-* ``Remove`` button: Removes the selected folder from the sync process.  This 
-  button is used when you want to synchronize only a few folders and not the 
-  root folder. 
-
-* ``Storage Usage`` field: Displays how much space your files are using on the 
-   ownCloud server.
-
-* ``Edit Ignored Files`` button: Launches the Ignored Files Editor.
-
-* ``Modify Account`` button: Use this to change your ownCloud server settings 
-  by launching the account setup wizard (see :doc:`accountsetup`).
-* ``Modify Account`` button: Use this to change your ownCloud server settings 
-  by launching the account setup wizard (see :doc:`accountsetup`).
-
-Adding a Folder
-^^^^^^^^^^^^^^^
-
-The ``Add Folder ...`` button enables you to add a new local folder to the 
-synchronization process.
+Click **Settings** in the right-click menu to see a summary of your ownCloud 
+account settings. This shows which ownCloud account you are connected to, your 
+quota status, and a window for managing your synchronization settings.
 
-To add a new folder:
+.. image:: images/client6.png
+   :alt: Account settings window
+   
+At the top of this windows are four tabs: your account settings, which is 
```

Comment:
Yes, agree - the documentation needs to allow for when a user actually has multi-account set up.